frontend/simple/sass/_groupincome.scss

```diff
@@ -74,3 +74,22 @@
   background-color: #1fc8db;
   border-radius: 20px;
 }
+
+//--------------------------------------
+// Modifiers
+//--------------------------------------
+.is-danger {
+  color: $red;
+}
+.is-success {
+  color: $green;
+}
+.is-warning {
+  color: $orange;
+}
+.is-light {
+  color: $grey;
+}
+.is-small {
+  font-size: $size-7;
+}
```

Comment:
Sorta. Bulma does a poor job of documenting these features in the readme but as far as I can tell, they only apply to very specific components like buttons and not plain text. I can make a note about that.